client/src/pages/TaskFormPage/TaskFormPage.js

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import styles from "./TaskFormPage.module.css";
+import { InputBox } from "../../components/InputBox/InputBox";
+import { CredentialsButton } from "../../components/CredentialsButton/CredentialsButton.js";
+import CloseButton from "react-bootstrap/CloseButton";
+
+const TaskFormPage = () => {
```

Comment:
![image](https://user-images.githubusercontent.com/24628243/159597026-fa60a2e1-4c4e-48f6-87be-7dea813ea6b4.png)
For the page itself, I believe this should be centered no?

Reply:
As of right now, everything is just on the top left of the screen